passkey-api/introduction.mdx

```diff
@@ -11,11 +11,69 @@ payment transactions.
 <Tip>
 Note that the Passkey API is not designed to be a full authentication solution — it's only intended to be used together with another auth system.
 
-If you need a full, **open-source** auth solution, check out [Hanko Auth](/introduction)
+If you need a full, **open-source** auth solution, check out [Hanko Auth](/introduction).
```

Comment:
Unrelated to the added dot (.), maybe we should rewrite this or at least change the emphasis from "open-source" to "full"

Otherwise it could be mistaken to mean that the Passkey API isn't open source, but Hanko is

...when it's really all about the fact that Hanko is the **full** auth and user management solution (and **not** about the fact that it's open-source)